src/modules/esl-panel-group/core/esl-panel-group.ts

```diff
@@ -31,8 +31,8 @@ export class ESLPanelGroup extends ESLBaseElement {
   @attr({defaultValue: ''}) public modeClsTarget: string;
   /** Class(es) to be added during animation ('animate' by default) */
   @attr({defaultValue: 'animate'}) public animationClass: string;
-  /** Time to clear animation common params (max-height style + classes) ('auto' by default) */
-  @attr({defaultValue: 'auto'}) public fallbackDuration: number | 'auto';
+  /** Time to clear animation common params (max-height style + classes) */
```

Comment:
Here is the correct definition.
The value 'auto' disables fallback, and it is a default fo component.